### ISSUE TITLE
Typo in the code?

### ISSUE THREAD

**Issue by xiaogengen-007** [MIAGroupUT/IDIR#4]
Hi,

Is there a typo in the following code block?

> regularizers.py lines 77-81

`def compute_bending_energy(input_coords, output, batch_size=None)`
`loss += (
        2 * torch.mean(dx_xyz[:, :, 1])
        + 2 * torch.mean(dx_xyz[:, :, 2])
        + torch.mean(dy_xyz[:, :, 2])
    )` 

Thanks for the great work and sharing the code!
